tutorials/simple-demo/backend/src/ManualApprove/assets/fieldConfig.json

```diff
@@ -0,0 +1,52 @@
+{
+  "invoice_date": {
+    "type": "date",
+    "display": "Fakturadato",
+    "confidenceLevels": { "automated": 0.98, "highest": 0.97, "high": 0.9, "low": 0.5 }
```

Comment:
Not that it matters that much, but I think 'highest' is a weird threshold, maybe rename it to 'very_high'? (I mean 'automated' is the 'highest' confidence threshold isn't it?)